### PR TITLE
[Enhancement] Integrate delta_writer_commit span into load_channel span

### LINKED ISSUE #7434: [Enhancement] Integrate delta_writer_commit span into load_channel span
## Enhancement
Currently, delta_writer_commit span runs in tablet_writer_add_chunk rpc handler, which is in bthread environment, and aync_delta_writer is used, so thread may get switched, as a result delta_writer_commit didn't `catch` load_channel span automatically, so we need to set delta_writer_commit span's parent explicitly.

### PR DESCRIPTION
**Pull request by zombee0**
Signed-off-by: zombee0 <flylucas_10@163.com>

## What type of PR is this：
- [x] enhancement

## Which issues of this PR fixes ：
Fixes #7434 

## Problem Summary(Required) ：
Integrate delta_writer_commit span into load_channel span

### CODE REVIEW

File: be/src/runtime/load_channel.h [hunk 1]
```diff
@@ -85,6 +85,8 @@ class LoadChannel : public RefCountedThreadSafe<LoadChannel> {
 
     MemTracker* mem_tracker() { return _mem_tracker.get(); }
 
+    std::string get_span() { return Tracer::to_trace_parent(_span->GetContext()); }
```
Comment (decster) [inline be/src/runtime/load_channel.h:88]: why not just return _span, trace_parent is not need in rpc or http request.

Comment (zombee0) [reply be/src/runtime/load_channel.h:88]: Thank you so much, that will be better.

**Review by decster — DISMISSED** [review]

**Comment by decster** [comment]
run starrocks_be_unittest

**Comment by decster** [comment]
run starrocks_clang-format

**Review by decster — APPROVED** [review]

**Review by chaoyli — APPROVED** [review]

**Pull request merged**